fix: pipe the whole generate-spec prompt through stdin in the cursor provider (#2959)
Redocly/redocly-cli: .changeset/generate-spec-cursor-stdin.md

---
'@redocly/cli': patch
---

> Fixed the `cursor` AI provider of the `generate-spec` command sending only the instructions to the model: the current Cursor CLI ignores piped stdin when a prompt argument is present, so the operation to refine never reached it.

Reviewer: ```suggestion
Fixed an issue where the `cursor` AI provider of the `generate-spec` command sent only the instructions to the model and the operation to refine never reached it.
```
